source/backups.rst

```diff
@@ -166,6 +166,35 @@ configures a private volume for filesystem-type storage:
    take effect after applying the updated ``deploy/cr.yaml`` file with
    ``kubectl``.
 
+.. _backups-compression:
+
+Enabling compression for backups
+--------------------------------
+
+Starting from the Percona XtraDB Cluster Operator 1.5.0, it is possible to
```

Comment:
I think all our documentation only about the latest version, we can put many "in version" phases near each phrase
```suggestion
It is possible to
```